src/Microsoft.Health.Dicom.Core/Features/Operations/DicomDurableFunctionsHttpClient.cs

```diff
@@ -0,0 +1,75 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Globalization;
+using System.Net;
+using System.Net.Http;
+using System.Net.Mime;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Dicom.Core.Configs;
+using Microsoft.Health.Dicom.Core.Messages.Operations;
+using Microsoft.Health.Dicom.Core.Models.Operations;
+using Microsoft.Net.Http.Headers;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Dicom.Core.Features.Operations
+{
+    internal class DicomDurableFunctionsHttpClient : IDicomOperationsClient
```

Comment:
Perhaps. Although this *is* core logic. Just because it is web based doesn't mean it is violating separation of concerns. Of course, I suppose we could have different implementations of the `IDicomOperationsClient` live in different assemblies specific to their technology, like `Http` or `gRPC`. However, `HttpClient` is part of the base class libraries and we'd leverage it regardless of whether our app was hosted in ASP.NET Core or running as a console app.

Reply:
If we did split it out, we'd probably have something like the following:
- `Microsoft.Health.Dicom.Operations.Abstractions`
    - Defines data models and interfaces for interacting with the DICOM operations
- `Microsoft.Health.Dicom.[Operations.]Functions`
    - Defines the actual Azure Functions and depends on `Microsoft.Health.Dicom.Operations.Abstractions`
- `Microsoft.Health.Dicom.[Operations.]Functions.Client`
    - Defines the HTTP client implementation for communicating with our Azure Functions and depends on `Microsoft.Health.Dicom.Operations.Abstractions`

Reply:
I've moved the client to its own project like SQL-stuff